front/src/modules/trainschedule/components/Timetable/PacedTrain/PacedTrainItem.tsx

```diff
@@ -106,6 +119,24 @@ const PacedTrainItem = ({
     }
   };
 
+  async function deleteExceptions() {
+    const formattedPacedTrain = formatPacedTrainWithDetailsToPacedTrainPayload(pacedTrain);
+
+    await storePacedTrain(
+      pacedTrain.id,
+      {
+        ...formattedPacedTrain,
+        exceptions: [],
+      },
```

Comment:
Better give it a name than having a plain object in the middle of the function many arguments

```suggestion
    const updatedPacedTrainPayload = {
      ...formatPacedTrainWithDetailsToPacedTrainPayload(pacedTrain),
      exceptions: [],
      }

    await storePacedTrain(
      pacedTrain.id,
      formattedPacedTrain,
```

Reply:
Done

Reply:
Done